Formula/ooniprobe.rb

```diff
@@ -15,6 +15,9 @@ class Ooniprobe < Formula
   depends_on "go" => :build
 
   def install
+    # fetch resources
+    system "go", "run", "./internal/cmd/getresources"
```

Comment:
Is this downloading unversioned unchecksummed things?

Reply:
oh yes, maybe I can install the resource. Let me give that a try.

Reply:
The download is both versioned and checksummed. The code that implements it is at https://github.com/ooni/probe-cli/blob/master/internal/cmd/getresources/getresources.go#L46 and the information about what to download is at https://github.com/ooni/probe-cli/blob/master/internal/engine/resources/assets.go. (Thank you @SMillerDev and @chenrui333!)